packages/smooth_app/lib/pages/product/common/product_query_page.dart

```diff
@@ -203,6 +203,7 @@ class _ProductQueryPageState extends State<ProductQueryPage> {
                                     style: themeData.textTheme.subtitle1!
                                         .copyWith(color: widget.mainColor)),
                                 style: TextButton.styleFrom(
+                                  primary: widget.mainColor,
```

Comment:
Lets rather also just change the splash color 
https://stackoverflow.com/a/64380957/13313941

Reply:
Hi @M123-dev ,
I tried adding overlay color, here is the updated screenshot.
<img width="264" alt="image" src="https://user-images.githubusercontent.com/47862474/161493238-d3eae749-1e91-4183-a771-2f861dcd419c.png">

Reply:
I meant for the back button, not for the filter, so that they both have the same splash but are not completely filled the whole time

Reply:
I think the current screenshot is misleading, so here is a video of it
https://user-images.githubusercontent.com/47862474/161748967-8d3c2992-6c70-440c-bfec-f6adfbca65a6.mp4

.